task/script.js

```diff
@@ -1,9 +1,285 @@
 const API_BASE_URL = window.API_BASE_URL;
+const SKILLS_API = 'https://services.realdevsquad.com/skilltree/v1/skills';
+
 const suggestedUsers = [];
 const allUser = [];
+
+const params = new URLSearchParams(window.location.search);
+const isDev = params.get('dev') === 'true';
+
+const featureUrlField = document
+  .getElementById('featureUrl')
+  .closest('.inputBox');
+
+if (isDev && featureUrlField) {
+  featureUrlField.style.display = 'none';
+}
+const featureRadio = document.getElementById('feature')?.closest('.inputBox');
+
+if (isDev && featureRadio) {
+  featureRadio.style.display = 'none';
+}
+const taskLevelDiv = document
+  .querySelector('.inputBox label[for="taskLevel"]')
+  .closest('.inputBox');
+
+if (isDev && taskLevelDiv) {
+  taskLevelDiv.style.display = 'none';
+}
+const skillComponentDiv = document
+  .querySelector('.multi-select-button')
+  ?.closest('.inputBox');
+if (!isDev && skillComponentDiv) {
+  skillComponentDiv.style.display = 'none';
+}
```

Comment:
- instead of defining a variable and writing a condition what if we define all the variables first and then add the required checks? this will make it easy for others contributing to the codebase to skim the code. what do you think?

Reply:
> * instead of defining a variable and writing a condition what if we define all the variables first and then add the required checks? this will make it easy for others contributing to the codebase to skim the code. what do you think?

You are right, have updated accordingly.